src/containers/Member.js

```diff
@@ -148,6 +149,38 @@ const MemberPage = ({
   const title = titleText + siteMeta.name.separator + siteMeta.name.full
   const canonical = `${siteMeta.urlOrigin}${pathname}`
 
+  const getReadingTimeAndUnit = readingSec => {
+    const minutes = Math.floor(readingSec / 60)
+    if (minutes < 10000) {
+      return {
+        articleReadingTime: minutes,
+        articleReadingTimeUnit: READING_TIME_UNIT.minute,
+      }
+    } else if (minutes < 1000 * 60) {
+      const hours = Math.floor(minutes / 60)
+      return {
+        articleReadingTime: hours,
+        articleReadingTimeUnit: READING_TIME_UNIT.hour,
+      }
+    } else {
+      const days = Math.floor(minutes / (60 * 24))
+      return {
+        articleReadingTime: days,
+        articleReadingTimeUnit: READING_TIME_UNIT.day,
+      }
+    }
+  }
+
+  // TODO: remove after get data from api
+  // ?hideInfo=true&articleReadCount=1000&readingSec=123
+  const param = new URLSearchParams(search)
```

Comment:
Just a bonus:
we could also use [`useParams`](https://v5.reactrouter.com/web/api/Hooks/useparams) to get url params by:
```
const { hideInfo } = useParams()